### ISSUE TITLE
Hotspots editing crashes the game

### ISSUE THREAD

**Issue by andresedu1996** [paulov-t/FrostbiteModdingTool#80]
Seems that kit hotspot editing (crashes the game)... Also if possible could it be possible to export the hotspot file in a .hotspot file format

**Comment by andresedu1996** [comment]
It is a file format used on hotspots instead of a JSON I can share one if you'd like, it is fairly similar and you can open it with VS.

I mentioned it because it might make it easier to transfer modded hotspot files data from other tools like FET to FMT or vice versa.

**Comment by moT-D** [comment]
Just here to confirm my crashing too. I changed the values (x,y,z,w) on a kit hotspot for various things like team logo placement, and now it crashes when I load my career. Everything was working before I made that change.

**Comment by moT-D** [comment]
> > It is a file format used on hotspots instead of a JSON I can share one if you'd like, it is fairly similar and you can open it with VS.
> > I mentioned it because it might make it easier to transfer modded hotspot files data from other tools like FET to FMT or vice versa.
> 
> There's no such thing as a hotspot file. Thats why I asked. Sounds like something that FET came up with. Which means I would have to hack into FET code again. 👎

I can give you a hotspot file if you'd like. You're right though, it's just called a hotspot file. The actual format is .json

**Comment by andresedu1996** [comment]
> > It is a file format used on hotspots instead of a JSON I can share one if you'd like, it is fairly similar and you can open it with VS.
> > I mentioned it because it might make it easier to transfer modded hotspot files data from other tools like FET to FMT or vice versa.
> 
> There's no such thing as a hotspot file. Thats why I asked. Sounds like something that FET came up with. Which means I would have to hack into FET code again. 👎

Yeah, you're right after analyzing it is definitely a JSON file. Maybe just something they made up

**Comment by DavoTriskele** [comment]
Hi guys
I also encountered the same problem. Changing even a single value compared to the original coordinates causes the game to crash
